src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace CryptoAxus.Domain.Collections;
+
+[BsonCollection("NFTCopies")]
+public class NFTCopiesDocument : BaseDocument
+{
+    [BsonElement(elementName: "nftId", Order = 2)]
+    public string NftId { get; set; }
+
+    [BsonElement(elementName: "owner", Order = 3)]
+    public string Owner { get; set; }
+
+    [BsonElement(elementName: "ownerName", Order = 4)]
+    public string OwnerName { get; set; }
+
+    [BsonElement(elementName: "quantity", Order = 5)]
+    public int Quantity { get; set; }
+
+    [BsonElement(elementName: "isCreator", Order = 6)]
+    public bool IsCreator { get; set; }
+
+    [BsonElement(elementName: "marketTokenId", Order = 7)]
+    public int MarketTokenId { get; set; }
+
+    public NFTCopiesDocument() : base()
+    {
+    }
+
+    public NFTCopiesDocument(ObjectId id, 
+                          string nftId,
+                          string owner,
+                          int quantity,
```

Comment:
No parameter for `OwnerName` field added.

Reply:
Resolved